### ISSUE TITLE
Reviews

### ISSUE THREAD

**Issue by murtosl** [murtosl/HistoryOfFlash_4662W#3]
Lucy: This project chronicles the way Flash adapted to a changing web environment over the years. I think this works well because it provides opportunities to discuss contentions and contemporary writing projects in a way that is fluid and natural.

I think your pages do a good job of building on each other and feeling very connected. Each page describes a different way that Flash changed to create new uses or stronger usability. You cover a lot of information and you organize it well; the only thing I would suggest is seeing where you can introduce more analysis based on our readings. You are doing some of that already but more might help break up the density of the straight factual information.

The hierarchy is good. I like the large, recognizable header on all the pages. I think the Adobe red is a little on-the-nose but it looks very professional. (I’m not as big a fan of the shadowed letters; the shadows don’t look consistent anyway.) The one area I felt struggled with hierarchy was the “A Mobile Future” page. I think the italic lines at the top and bottom of the page are important to understanding what it going on in the page, but they are hidden with a small font compared to the other things on the page. In addition, on both that page and the intro page the text is larger than the text on other pages. I assume it is the same font but I can’t tell because of the size difference. For consistency, I would keep all the paragraph-type text the same size. The navigation is very thorough and professional. There is a large nav bar at the top as well as links at the bottom of each page to the previous and next pages. My only concern is that the links at the bottom are a little squashed and might look better with more space.

You utilize a lot of helpful sources and resources. I think you do a fantastic job organizing such a large amount of information and presenting it in an accessible way. You cover contentions and contemporary writing projects. Like I said before, my suggestion might be to do more analysis using Haas’s simple replacement theory and so on. Bringing in more analysis would help provide a stronger scholarly context for the development of Flash. But otherwise I think you did a great job historicizing.

On the very last page, you could include a link at the bottom that goes back to the introduction (then it would really go full circle ;) ).

For quotes, you’ll want the quotations marks to go inside the period at the end and before the citation. E.G. “Old technology” (Haas 1). I noticed a few of these on the Usability page. (Unless you’re using a citation format I’m not familiar with.)
# 

John: Sarah’s website takes us on a very thorough and well-paced journey through the history of Flash, from its beginnings as FutureSplash, to Apple’s refusing to run it on the iPhone, effectively putting a limit on the ultimate possibilities of the plug-in.
From the beginning I got a very clear feeling about who the clashing parties were in the determination of whether Flash would become the keystone plug-in of our time. She does a good job of backgrounding plug-ins in general, reminding us of a time when it wasn’t even sure if the technology itself would catch on or ever be viable. But when it comes to Flash, there is a great assessment of the watershed moments in its history. Finding an article which represent a whole side of the argument is really effective at creating a context for the historicizing of that moment.
I also liked the quotation of Haas’s article on Historicizing, and think those are applied very well. There are good visual breaks in the course of the site, with a page containing a moment when Adobe was looking toward the future of their product, one which they couldn’t make come to be. The images in the series of pages work very well with the content of their pages, especially the Homestar Runner video. 
Code Review
The use of a double-decker and staggered nav-bar is an impressive feature, but I can’t decide what I think of it visually. On some pages it makes the top a little busy, and I’m tempted to think it might work better on the side. Nonetheless, I think it is good that there are links to each page on every page.
On the The Future That Wasn’t page, the content runs up to the image, and it does look a little cramped.
I do like that the links lead to new tabs, too. Something I wish I had done more of on my site.
# 

Cindy:  General thoughts:

The overall argument on this site is that the Flash plugin played a role fulfilling an essential need when it was first developed. As the internet evolved and became more complex, Flash also evolved and took on more roles. However as more open-source solutions are used on the internet, Flash has been replaced by newer technology.

The provided content all supports the argument clearly and thoroughly, so I don’t see any need to change anything here.

The visual appearance of the page is very nice. It looks professional and is very easy to navigate, although I just noticed that there are links in the footer. You may want to put the links above the “Created by” part and make it more obvious that these are links. Also, it is not obvious that there are two different links there, so you may want to do something to make that more obvious.

Code Review and some details

There are a few minor formatting and phrasing issues.

The text on the page with the Apple box butts up against the box in places. It would look more pleasing if there was some padding around the box.

On the “Design Revolution” page, the paragraph that begins with “In August 1999” I would change such that it reads “ActionScript and ‘support…’ were introduced to Flash.” Using “as well as” is a bit awkward. Also consider adding a word before “an animation”. The way it is written makes it sound like Flash doesn’t do animations. The next part of the sentence makes it clear that is not the case, but the reader has to go back and think about it. I like to avoid doing this to my readers if I can. Also in this section, the block quote that starts with “Macromedia’s” doesn’t begin with a quotation mark but ends with one. I found myself trying to go back and figure out if there was a quotation within the quotation or if the block was an extension of something that started earlier in the paragraph.

In the “Wars on Incompatibility” section, replace “It” with something more specific. I wasn’t sure exactly what time you were referring to.

In the “Web as a Canvas” section, I’m not sure “emerging” is the best verb to use in this context. If it isn’t part of a quote, I would consider using something like “immersing”. Also in this section, I’m confused about the quotation. I’m guessing the quotation from Ankerson is incomplete and you wanted to add the “designers embraced” part to complete the sentence in your own words. In that case I think the sentence should look something like this: “Flash represented a return to a creative core that [designers embraced].” (Ankerson, 277) I’m not entirely sure, so you may want to ask for other opinions on this.
# 

Prof:

See Drive

**Comment by murtosl** [comment]
Revision due Dec 4th

**Comment by murtosl** [comment]
Post Content for this chunk: 

I took everyone's suggestions and notes into considerations and filled out issues for each task I'm going to tackle. I have separated these issues into design issues and content issues. I have more design issues than content issues. https://github.com/murtosl/HistoryOfFlash_4662W/issues

I'm going to be gone for two whole weekends, so those weeks are going be a slight wash, and I have a large final I need to start writing for another class. So this is getting done this weekend, which I thought I was going to have to do anyways before the deadline was changed. 

My timeline is to get all the design tweaks done Saturday, and hopefully some paper revising.  Sunday will be content revision in the morning, and a proofreading revision in the late afternoon. I will do additional proofreading and revision Monday and have my project ready Friday morning before I leave. If I find I did not meet my goals, I have my wiggle room in the remaining weeks, but I would like to just wrap this up and a call it good. I will probably end up doing a final proofreading  right before the 4th.

So hopefully I'm finished with my paper by the time you read my timeline.
